jdk/src/solaris/bin/zero/jvm.cfg

```diff
@@ -1,4 +1,7 @@
-# Copyright (c) 2009, 2013, Oracle and/or its affiliates. All rights reserved.
+#
+# (c) Copyright IBM Corp. 2003, 2017 All Rights Reserved
```

Comment:
Please set 2018

Reply:
Fixed - Good spot - I read 2017 and thought "Yeah that's up to date" ... #JanuaryFever

Reply:
Done